Dockerfile.worker

```diff
@@ -14,6 +14,11 @@ COPY data_models/dist/data-refinery-models-* data_models/
 # Get the latest version from the dist directory.
 RUN pip install data_models/$(ls data_models | sort -r | head -1)
 
+COPY data_models/dist/data-refinery-models-* data_models/
+
+# Get the latest version from the dist directory.
+RUN pip install data_models/$(ls data_models | sort -r | head -1)
```

Comment:
Actually `ls` command already sorts the filenames alphabetically, so the command can be more compact: 
`ls -r1 | head -1` 
or 
`ls -1 | tail -1`